tests/sanity/sanity_test.go

```diff
@@ -62,8 +62,9 @@ var _ = Describe("AWS FSx for OpenZFS CSI Driver", func() {
 		StagingPath:    stagePath,
 		TestVolumeSize: 2000 * util.GiB,
 		TestVolumeParameters: map[string]string{
-			"volumeType":      "filesystem",
-			"skipFinalBackup": "true",
+			"volumeType":                "filesystem",
+			"skipFinalBackupOnDeletion": "true",
+			"optionsOnDeletion":         "[DELETE_CHILD_VOLUMES_AND_SNAPSHOTS]",
```

Comment:
Why was optionsOnDeletion added here? I believe I only added variables that if missing, would throw errors.

Reply:
Yeah this was a mistake on my part - I very briefly thought we would make this parameter required, then forgot to remove this when I realized we weren't going to.